### DISCUSSION TITLE
I'm having difficulty locating the option to disable YAML frontmatter reformatting. Could you please help me find it?

### DISCUSSION THREAD

**Discussion by jwliles** [A-F-V/obsidian-arcana#48] [General]
The plugin is altering the YAML frontmatter in my notes, and my attempts to revert the changes are being overridden. Is this the plugin's default behavior, or is there a setting I'm missing?

**Comment by A-F-V** [comment]
So when obsidian wants to get some data from the front matter, it standardises the formatting. This is because the front matter is not something that can have a user defined format and it just regresses to YAML formatting. 

Now at the moment darwin gets all the tags in all the notes by visiting each note which causes a reformat. 

Now I am going to revert back to an early version until tomorrow and then will see if there is a way that not all notes have to be visited (because maybe tags are cached?) and hence trigger a reformat. However, whenever tags are added there is no way to avoid the whole yaml being reformatted sadly

**Reply by jwliles** [reply]
Alright, I understand and appreciate your feedback. To maintain consistency in my vault, I use the Linter plugin to ensure stylistic standardization. I will make the necessary adjustments to align with the reformatted YAML. Thank you for getting back to me.

**Reply by A-F-V** [reply]
@jwliles Do you mind sending an example of the yaml before and after that you are observing?

**Reply by jwliles** [reply]
I apologize for the delay in my reply. I did not receive a notification of your response.

Regarding the frontmatter issue, it has been mostly resolved. I have decided to use a format that closely resembles what the plugin has been formatting to. However, there may be times when the plugin removes certain escape characters and alters the formatting.

**Reply by A-F-V** [reply]
Try text in quotations. For example, I have a template that has `created: {{date}}`. However, yaml processing will get confused and so you should instead write `created: "{{dated}}"` to avoid it getting reformatted upon frontmatter handling

**Reply by zanodor** [reply]
This reformatting is pretty annoying because some template files to do with Zotero (nunjucks?) are completely altered. Not to mention having 10k+ changes in the repo and I forgot to commit changes before launching the plugin. Meh...

> tags are cached

You can get the metadatacache with Obsidian API without altering the files. 
An example:
https://zachyoung.dev/posts/templater-multi-select-suggester
